harper-core/src/word_metadata.rs

```diff
@@ -113,6 +113,41 @@ impl WordMetadata {
         )
     }
 
+    /// Checks if the word is a homophone
+    /// This is based off of metadata, so it is not guaranteed to be correct
+    pub fn is_homophone(&self) -> bool {
+        [
```

Comment:
I'd like to backtrack something I said earlier. Change the name of this function to `is_likely_homophone` to indicate that, unlike the other `is_&` functions, this is not definitive.

Reply:
I would also like to point out that these are homographs, not homophones